src/styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  border: 1px solid red;
+  text-align: center;
+  margin: 0;
+  display: block;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+html {
+  height: 100vh;
+}
```

Comment:
I wasn't familiar with this - let's remember to come back and take another look at how it's working